zuul.d/projects.yaml

```diff
@@ -1,7 +1,21 @@
 ---
+- job:
+    name: openstack-services-content-provider
+    parent: cifmw-tcib-base
+    nodeset: centos-stream-9-vexxhost
+    description: |
+      Zuul job to build rpms for openstack projects from opendev, tcib and
+      os-net-config from github. It also builds openstack services container
+      for openstack projects.
+
 - project:
     name: os-net-config/os-net-config
     github-check:
       jobs:
         - tox-linters:
             nodeset: rdo-centos-9-stream
+        - openstack-services-content-provider
+        - podified-multinode-edpm-deployment-crc:
+            override-checkout: main
```

Comment:
could do with files: here i think, looking at the repo https://github.com/os-net-config/os-net-config 
for example we don't want this to run for "doc" or /etc/os-net-config/samples or releasenotes

probably like 

```
files:
  - ^os_net_config/*.py
```